src/fhir/1_Condition_DCW_MP_0.json

```diff
@@ -0,0 +1,39 @@
+{
+  "resourceType": "Condition",
+  "id": "condition-DCW-MP-0",
+  "clinicalStatus": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code": "active",
+        "display": "Active"
+      }
+    ],
+    "text": "Active"
+  },
+  "verificationStatus": {
+    "coding": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code": "confirmed",
+        "display": "Confirmed"
+      }
+    ],
+    "text": "Confirmed"
+  },
+  "code": {
+    "coding": [
+      {
+        "system": "http://www.cdc.gov/nchs/icd/icd10cm.htm",
+        "code": "F01.5",
+        "display": "Vascular Dementia, unspecified severity"
+      }
+    ],
+    "text": "Vascular Dementia, unspecified severity"
+  },
+  "subject": {
+    "reference": "Patient/patient-dcw-mp"
```

Comment:
@mcjustin  not sure if this is referencing subject `patient-dcw-ms` or `patient-dcw-pm` as I don't see a patient with an ID of `patient-dcw-mp`